src/app/api/cms/messages/clear_messages.ts

```diff
@@ -0,0 +1,73 @@
+import ActiveITSClient from "@entities/ActiveITSClient";
+import ActiveITSDB from "@entities/ActiveITSDB";
+import EmailSender from "@entities/EmailSender";
+import { SaveHistory } from "@entities/SaveHistory";
+import * as env from "@utils/config.constant";
```

Comment:
`import * as constants` would make more sense to me since you're not importing environment variables from here